### PR TITLE
CAD-1310 JSON configs for the TX generator

### PR DESCRIPTION
**Pull request by MarcFontaine**
This PR adds JSON based tx-generator scripts.
The example script in `benchmarks/shelley3pools/tx-generator-script.json` is started with
```
era=json make cabal
```

### CODE REVIEW

File: benchmarks/shelley3pools/tx-generator-script.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+[
```
Comment (deepfire) [inline benchmarks/shelley3pools/tx-generator-script.json:1]: @MarcFontaine, can we improve the density, by reformatting this very JSON as one line per command?

Comment (deepfire) [reply benchmarks/shelley3pools/tx-generator-script.json:1]: ..unless the command itself is large, of course.

Comment (MarcFontaine) [reply benchmarks/shelley3pools/tx-generator-script.json:1]: Sure. as you please.
This is how it came out of the pretty printer. I don't think there is a pretty-printer setting for this.
So I'll edit it manually

Comment (MarcFontaine) [reply benchmarks/shelley3pools/tx-generator-script.json:1]: Best would be to add a THIS IS A GENERATED FILE! DO NOT EDIT ! comment

**Review by deepfire — APPROVED** [review]
LGTM, thank you @MarcFontaine!

**Pull request merged**